PED/IsPedInjured.md

```diff
@@ -9,11 +9,19 @@ BOOL IS_PED_INJURED(Ped ped);
 ```
 
 ```
-Gets a value indicating whether this ped's health is below its injured threshold.  
-The default threshold is 100.  
+Get 1 or false if PED is Injured.
 ```
 
 ## Parameters
 * **ped**: 
 
 ## Return value
+1 or false
```

Comment:
This function returns a boolean, therefor the return value should either be `true` or `false`. Not trying to be picky, just trying to keep things consistent :-)